### PR TITLE
Creating the changes for qti backend for default backend

### PR DESCRIPTION
**Pull request by mohitmundhragithub**

### CODE REVIEW

**Comment by mohitmundhragithub** [comment]
Please merge this only after PR#743

**Comment by mohitmundhragithub** [comment]
@anhappdev, can you please review the changes?

File: mobile_back_qti/cpp/backend_qti/settings/qti_settings_default_dsp.pbtxt [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  custom_setting {
+    id: "bg_load"
+    value: "true"
+  }
+  custom_setting {
+    id: "use_ion_buffer"
+    value: "false"
+  }
+  model_path: "https://github.com/mlcommons/mobile_models/releases/download/v3.0-snpe/mobilenet_edgetpu_224_1.0_htp.dlc"
+  model_checksum: "2e7c4d33b480b5566bdf05e1204b6152"
```
Comment (anhappdev) [inline mobile_back_qti/cpp/backend_qti/settings/qti_settings_default_dsp.pbtxt:36]: Can you please migrate all the old `benchmark_setting` to the new schema with `delegate_choice`?

File: mobile_back_qti/cpp/backend_qti/settings/qti_settings_sd7pg2.pbtxt [hunk 2]
```diff
@@ -10,24 +10,36 @@ common_setting {
...
-  model_checksum: "2e7c4d33b480b5566bdf05e1204b6152"
-  single_stream_expected_latency_ns: 600000
+  delegate_choice: {
+    priority: 1
+    delegate_name: "SNPE_DSP"
+    accelerator_name: "snpe_dsp"
+    accelerator_desc: "HTP"
+    model_path: "https://github.com/mlcommons/mobile_models/releases/download/v3.0-snpe/mobilenet_edgetpu_224_1.0_htp.dlc"
+    model_checksum: "2e7c4d33b480b5566bdf05e1204b6152"
+    single_stream_expected_latency_ns: 600000
```
Comment (anhappdev) [inline mobile_back_qti/cpp/backend_qti/settings/qti_settings_sd7pg2.pbtxt:36]: `single_stream_expected_latency_ns` is not a field of `delegate_choice` but `benchmark_setting`, you need to move it one level up.

Files to check:

- `qti_settings_sd7pg2.pbtxt`
- `qti_settings_sd8cxg3.pbtxt`
- `qti_settings_sd8g1.pbtxt`
- `qti_settings_sd8g2.pbtxt`
- `qti_settings_sd8pg1.pbtxt`

**Comment by mohitmundhragithub** [comment]
@anhappdev  please review.

**Review by anhappdev — APPROVED** [review]

**Pull request merged**